jobs/Scheduler.js

```diff
@@ -382,8 +387,7 @@ class Scheduler {
         .runAt(jobName,
           type,
           config.scheduler.jobs.reschedule_delay,
-          jobData,
-          CONST.SYSTEM_USER
+          jobData
```

Comment:
Any particular reason why we are not passing `user` param here?

Reply:
The method `runAt` never had user param. Was passing it wrongly here, so removed it.